saleor/graphql/app/tests/mutations/cassettes/test_app_fetch_manifest/test_fetch_manifest_fail_when_app_with_same_identifier_already_installed.yaml

```diff
@@ -0,0 +1,60 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate, br
+      Connection:
+      - keep-alive
+      User-Agent:
+      - python-requests/2.31.0
+    method: GET
+    uri: http://localhost:3000/api/manifest
+  response:
+    body:
+      string: !!binary |
```

Comment:
I see this is gzip encoded and I can decode it, but it would be great to just have JSON here, for reference to see how app response looks like and maintainability of such test.

Reply:
I managed to fix it and left a hint in our notion tutorial.